app/src/main/java/com/duckduckgo/app/browser/omnibar/OmnibarLayout.kt

```diff
@@ -85,15 +84,14 @@ import com.duckduckgo.common.utils.extensions.replaceTextChangedListener
 import com.duckduckgo.common.utils.text.TextChangedWatcher
 import com.duckduckgo.di.scopes.FragmentScope
 import com.google.android.material.appbar.AppBarLayout
-import dagger.android.support.AndroidSupportInjection
 import javax.inject.Inject
 import kotlinx.coroutines.flow.collectLatest
 import kotlinx.coroutines.flow.map
 import kotlinx.coroutines.launch
 import timber.log.Timber
 
 @InjectWith(FragmentScope::class)
-class OmnibarLayout @JvmOverloads constructor(
+open class OmnibarLayout @JvmOverloads constructor(
```

Comment:
there are now two omnibars that looks different and behave differently on scroll. however, the buttons and other ui elements are exactly the same. by creating an open class we can then use a `ScrollingOmnibarLayout` (this replaces the current omnbiar in Production) and a `FadeOmnibarLayout` (the new and delightful one)